homeassistant/components/stream/worker.py

```diff
@@ -127,6 +127,15 @@ def diagnostics(self) -> Diagnostics:
 class StreamMuxer:
     """StreamMuxer re-packages video/audio packets for output."""
 
+    _segment_start_dts: int
+    _memory_file: BytesIO
+    _av_output: av.container.OutputContainer
+    _output_video_stream: av.video.VideoStream
+    _output_audio_stream: av.audio.AudioStream | None
+    # the following 2 member variables are used for Part formation
+    _memory_file_pos: int
+    _part_start_dts: int
```

Comment:
These are initialized in the `reset` method
https://github.com/home-assistant/core/blob/0eea3176d6b6bf871acc7a340f748af88615637e/homeassistant/components/stream/worker.py#L242-L259

directly after the class init
https://github.com/home-assistant/core/blob/0eea3176d6b6bf871acc7a340f748af88615637e/homeassistant/components/stream/worker.py#L612-L620

Moving these type assignments from the `__init__` method to the class avoid dealing with`None`.